CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Current
 -------
 
 ### Added:
+- [Add Uptime Status Metric](https://github.com/yahoo/fili/pull/518)
```

Comment:
Just by the title it is not obvious to me what this change added. So maybe one or more bulletin points followed by explanation could do better.